packages/cf-style-const/src/variables.js

```diff
@@ -4,6 +4,13 @@ const colorOffsetLight = 0.15;
 const colorOffsetDark = 0.07;
 
 export default {
+  breakpoints: {
+    mobile: '@media (min-width: 13.6em)',
```

Comment:
Let's deal with this later. This is a pretty large and overarching change, I want to understand why they are in ems in the first place.